wazuh/wazuh-kibana-app
# Fix statistics visualizations

Hi team!

This PR close #2602 

**Steps for testing**
1. Checkout the branch and start Kibana (If the environment use ODFE or X-PACK, check if the internal user has permission in the wazuh-* indices)
2. Let documents be generated for 15 minutes
3. Go to `Management -> statistics` and check the visualizations.
4. All visualizations must have data in all time intervals.
